deploy/chart/templates/controller.yaml

```diff
@@ -300,8 +304,10 @@ spec:
             - name: PRIVATE_CLOUD_TAG
               value: "true"
 {{- end }}
+{{- if .Values.deploy.withStorageController }}
```

Comment:
Is this environment enabled in the ACK cluster? I don't remember adding this configuration.

Reply:
Yes, current template of ACK csi-provisioner addon contains this too.